CLIENT-SPECIFICATION.md

```diff
@@ -59,28 +59,17 @@ tldr -l
 
 The first argument that does not start with a dash (`-`), MUST be considered the page name.
 
-In addition, page names MAY contain spaces (e.g. `git status`) - such page names MUST be transparently concatenated with dashes (`-`). For example, the page name:
-
-```
-git checkout
-```
-
-becomes this:
-
-```
-git-checkout
-```
+Page names MAY contain spaces (e.g. `git status`) - such page names MUST be transparently concatenated with dashes (`-`). For example, the page name `git checkout` becomes `git-checkout`. Page names MAY contain mixed capitalization - such page names MUST be transparently lowercased. For example, the page name `eyeD3` becomes `eyed3`.
```

Comment:
The dashse don't read well to me.  What about this?
```suggestion
Page names MAY contain spaces (e.g. `git status`), and such page names MUST be transparently concatenated with dashes (`-`). For example, the page name `git checkout` becomes `git-checkout`. Page names MAY contain mixed capitalization, and such page names MUST be transparently lowercased. For example, the page name `eyeD3` becomes `eyed3`.
```

Reply:
done